tests/test_node_differ.py

```diff
@@ -86,12 +110,22 @@ def test_nested_nodes_ref_temp_link():
     diff = get_diff(prev_wikitext, curr_wikitext, lang='en')
     assert expected_changes == nd.get_diff_count(diff)
 
-def test_nested_nodes_media_format():
-    # Known that this test fails right now -- see https://github.com/geohci/edit-types/issues/13
-    curr_wikitext = prev_wikitext.replace("[[File:Carl Aigen Fischmarkt.jpg|thumb|''Fischmarkt'' by Karl Aigen]]",
-                                          "[[File:Carl Aigen Fischmarkt.jpg|thumb|Fischmarkt by Karl Aigen<ref>A reference</ref>]]",
-                                          1)
-    expected_changes = {'Text Formatting':{'remove':1},'Media':{'change':1}, 'Reference':{'insert':1}}
-    diff = get_diff(prev_wikitext, curr_wikitext, lang='en')
-    print(diff)
-    assert expected_changes == nd.get_diff_count(diff)
\ No newline at end of file
+
+# def test_move_template():
```

Comment:
let's leave this test uncommented -- i think it should pass once this change is merged, right?

Reply:
It returns an empty dictionary

Reply:
ahh okay -- well if it's still not working after the other fixes, then can leave commented. thanks!

Reply:
I looked into this and I understand better now. The reason you're getting an empty diff is that instead of the tree differ seeing it as a template move, it sees it as a series of text changes. When I post-process the diff though, it determines the paragraph text didn't actually change and so removes that, leaving no changes. This is a bug in the tree differ but I'll have to think about whether / how to fix it.